git_cookbook.md

```diff
@@ -1,6 +1,22 @@
 # Git cookbook
 Handy recipes for git'n it done.
 
+
+#### Bash Completions
+Get tab-completion for all your git commands, including branch names!
+
+1. [Install homebrew](https://github.com/Homebrew/homebrew/wiki/Installation), if you haven't already.
+1. Install git and bash-completion via homebrew _(you must install git via homebrew for this to work)_
```

Comment:
Do we want to add steps for installation if you have installed git prior to homebrew?